ush/global_nest_patch.sh

```diff
@@ -0,0 +1,14 @@
+#! /usr/bin/env bash
+
+# shellcheck disable=SC2155,SC2312
+HOMEgfs=$(cd "$(dirname "$(readlink -f -n "${BASH_SOURCE[0]}" )" )/.." && pwd -P)
+declare -rx HOMEgfs
+
+## patch config.base
+sed -i -e 's#export BASE_CPLIC=.*#export BASE_CPLIC="/scratch2/BMC/wrfruc/Guoqing.Ge/ufs-ar/ICS/global-nest"#' -e 's/export DO_NEST="NO"/export DO_NEST="YES"/' $HOMEgfs/parm/config/gfs/config.base
+
+## patch build_ufs
+sed -i -e 's/FV3_GFS_v17_coupled_p8_ugwpv1"/FV3_GFS_v17_coupled_p8_ugwpv1,FV3_global_nest_v1"/' -e 's/^#MAKE_OPT+=/MAKE_OPT+=/' $HOMEgfs/sorc/build_ufs.sh
```

Comment:
Three problems here:

1. Relying on commented-out code is a bad practice.
2. There is no reason to keep FV3_global_nest_v1 out of the compiled CCPP_SUITES. Adding a suite has no negative effect (other than a slight increase in compile time).
3. Patching scripts makes code management more difficult.
4. The nested workflow does not need -DFASTER=ON, so it should not be hard-coded to that. Only the C768 needs it, mostly as a limitation of Hera's nodes. We should have the freedom to change the build options without having to edit them in two places.
5. Using a -f option in build_ufs.sh is simpler and better matches the current design.

Reply:
This is a temporary file to avoid manual edits. It will not be committed to the authoritative repo.

Yes, as long as EMC accepts a new CCPP option in build_ufs.sh, we don't need this patch or user's edits.

Reply:
Sure. I am fine with changing build_ufs.sh directly. I just stripped out all changes which will affect current GFS pre-v17 so as to make it ready for a PR to the EMC repo. But we can try your idea and modify if they have concerns. Thanks!

Reply:
Having a clean set of changes is more important than avoiding future discourse with EMC. It'll be difficult to review when everything is hidden behind sed commands in a ush/global_nest_patch.sh script. You need to remove that sedding script and make the changes in each individual file instead. It'll be both easier to maintain and easier to review.